scripts/pr-infra-compose.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+# This script leverages the host override docker-compose file to run the OSRD stack.
+set -e
+
+# Usage
+
+if [ "$#" -ne 3 ] ||
+    { [ "$3" != "up" ] && [ "$3" != "down" ]; }; then
+    echo "Usage: $0 pr-number osrd.backup (up | down)" >&2
+    exit 1
+fi
+
+# Change ports in needed files
+export PR_NB="pr-$1"
+
+if [ "$3" = "up" ]; then
+
+    # Load the docker instance
+    sed -i "s/{{pr-nb}}/${PR_NB}/g" ./docker/osrdyne-pr-tests.yml
+    docker compose \
+        -p "osrd-pr-tests" \
+        -f "docker/docker-compose.pr-tests.yml" \
+        up -d
+    git restore ./docker/osrdyne-pr-tests.yml
```

Comment:
Would it be a interesting modification to modify `osrdyne` so it does evaluate environment variables before deserializing the file? This way, we could avoid this weird dance of `sed` and `git restore`.

Reply:
I'm pretty sure it does that already 😆

Reply:
I didn’t know I could combine the two methods. I’ve updated the script accordingly. ;)

Thanks!